ibis/backends/flink/registry.py

```diff
@@ -0,0 +1,72 @@
+import ibis.expr.operations as ops
+from ibis.backends.base.sql.compiler import ExprTranslator
+from ibis.backends.base.sql.registry import helpers, literal_formatters
+from ibis.backends.base.sql.registry import (
+    operation_registry as base_operation_registry,
+)
+from ibis.backends.flink.utils import format_interval_literal_segments
+
+operation_registry = base_operation_registry.copy()
+
+
+def _count_star(translator: ExprTranslator, op: ops.Node) -> str:
+    return "count(*)"
+
+
+def _interval_literal_format(translator, op):
+    timedelta_segments = format_interval_literal_segments(op.value)
+    return "INTERVAL " + " ".join(timedelta_segments)
+
+
+_literal_formatters = literal_formatters.copy()
+_literal_formatters.update({'interval': _interval_literal_format})
+
+
+def _literal(translator, op):
+    """Return the expression as its literal value."""
+
+    dtype = op.output_dtype
+
+    if dtype.is_boolean():
+        typeclass = 'boolean'
+    elif dtype.is_string():
+        typeclass = 'string'
+    elif dtype.is_date():
+        typeclass = 'date'
+    elif dtype.is_numeric():
+        typeclass = 'number'
+    elif dtype.is_timestamp():
+        typeclass = 'timestamp'
+    elif dtype.is_interval():
+        typeclass = 'interval'
+    elif dtype.is_set():
+        typeclass = 'set'
```

Comment:
The `Set` type has been removed in the dev branch - `dtype.is_set()` no longer exists.

```suggestion
```